src/lib/modules/Identifiers/Identifiers.js

```diff
@@ -70,6 +169,29 @@ export const IdentifierRows = ({ includeSchemes, identifiers }) => {
   });
 };
 
+const createValueObject = (vocabularyIdentifiers, value, scheme, material) => {
```

Comment:
this is CDS specific data, it should be implemented in the overlay, where the custom vocab is added